spec.md

```diff
@@ -945,23 +1076,29 @@ message NodePublishVolumeRequest {
   // this capability. This is an OPTIONAL field.
   PublishVolumeInfo publish_volume_info = 3;
 
+  // The path to which the device was mounted by `NodePublishDevice`.
```

Comment:
I agree credentials are not required in here nor do I think they should be included in ControllerPublish. IMO commands that are to be called once and not once per workload do not need to include the user/workload credentials.  Otherwise it opens up consistency issues.